packages/dinero.js/src/api/__tests__/toFormat.test.ts

```diff
@@ -11,58 +12,164 @@ import {
 } from '../../../../../test/utils';
 
 describe('toFormat', () => {
+  const simpleFormatter = ({ amount, currency }) =>
```

Comment:
This is more explicit IMO.

```suggestion
  const stringFormatter = ({ amount, currency }) =>
```